generation/tasks/PlotSplittedQuantity.py

```diff
@@ -0,0 +1,209 @@
+    )
+    match = luigi.ListParameter(
+        default=[],
+        description='Additional filters to match for reducing the analysis objects in the YODA file'
+    )
+    unmatch = luigi.ListParameter(
+        default=[],
+        description='Additional filters to unmatch for reducing the analysis objects in the YODA file'
+    )
+    yrange = luigi.TupleParameter(
+        default=[0.0, 10],
+        significant=False,
+        description="Value range for the y-axis of the plot.",
+    )
+
+    splittings_conf_summary = luigi.DictParameter(
+        # default="zjet",
+        description="BINS identifier (predefined binning configuration in generation/framework/config.py) for all splittings. \
+             Will set parameter 'splittings'. Overwritten by --splittings-all."
+    )
+    splittings_summary = luigi.DictParameter(
+        default=None,
+        description="Splittings plot settings for all bins. Set via --splittings-conf-all from config, if None.",
+    )
+
+    exclude_params_req = {
+        "source_script",
+    }
+    exclude_params_req_get = {
+        "htcondor_remote_job",
+        "htcondor_accounting_group",
+        "htcondor_request_cpus",
+        "htcondor_universe",
+        "htcondor_docker_image",
+        "transfer_logs",
+        "local_scheduler",
+        "tolerance",
+        "acceptance",
+        "only_missing",
+    }
+
+    def workflow_requires(self):
+        req = super(PlotSplittedQuantity, self).workflow_requires()
+        req["Rivet"] = RivetMerge.req(self)
+        return req
+
+    def create_branch_map(self):
+        bm = dict()
+        for jobid, flp in enumerate(self.filter_label_pad_tuples):
+            match, unmatch, xlabel, ylabel = flp
+            bm[jobid] = dict(match=match, unmatch=unmatch, xlabel=xlabel, ylabel=ylabel)
+        return bm
+
+    def requires(self):
+        return self.workflow_requires()
+
+    def local_path(self, *path):
+        parts = (
+            (os.getenv("ANALYSIS_DATA_PATH"),)
+            + (
+                self.__class__.__name__,
+                str(self.mc_generator).lower(),
+                self.campaign,
+                self.mc_setting,
+            )
+            + path
+        )
+        return os.path.join(*parts)
+
+    def output(self):
+        out = dict()
+        out["summary"] = self.local_target(
+            "m-{match}-um-{unmatch}/".format(
+                match=self.branch_data["match"],
+                unmatch=self.branch_data["unmatch"],
+            )
+        )
+        return out
+
+    def run(self):
+        check_outdir(self.output())
+        print("=======================================================")
+        print("Starting summary plotting for quantity {} with YODA".format(self.branch_data["match"]))
+        print("=======================================================")
+
+        inputs = dict()
+        logger.debug("\n\nInput:\n{}\n\n".format(self.input()))
```

Comment:
This debug message is also printed in the localize input function, called below. Remove it here

Reply:
done